Automattic/sensei
# Add Enrolled and Completion Rate columns to Courses report

Resolves #6881

## Proposed Changes
* Add a Enrolled column to the course report
* Add a Completion column to the course report

## Testing Instructions
1. Go to sensei admin
2. Create a course
3. Assing two students (without completing the course)
4. Check if the enrolled column shows the value 2
5. Assing a third student and complete the course with them.
6. Check if the Completion rate is 33%

![image](https://github.com/Automattic/sensei/assets/38718/5667b12d-f43d-4d92-a615-d68cdaf05d42)


## Pre-Merge Checklist
